templates/partials/summary/list-summary.html

```diff
@@ -1,38 +1,37 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
-{% if list.list_items | length > 0 %}
-
-  {% if list.editable %}
-    {% set headers = ["Name of person", "Action"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person",
-      _("Change"),
-      _("Change details for {item_name}"),
-      _("Remove"),
-      _("Remove {item_name}")
-    ) %}
-  {% else %}
-    {% set headers = ["Name of person"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person"
-    ) %}
-  {% endif %}
-
-  {{ onsSummary({
-    "withinQuestion": true,
-    "summaries": [
-      {
-        "groups": [
-          {
-            "headers": headers,
-            "rows": rows,
-          }
-        ]
-      }
-    ],
-    "classes": "u-mt-s u-mb-m"
-  }) }}
-
+{% if list.editable %}
+  {% set headers = ["Name of person", "Action"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person",
+    _("Change"),
+    _("Change details for {item_name}"),
+    _("Remove"),
+    _("Remove {item_name}")
+  ) %}
+{% else %}
+  {% set headers = ["Name of person"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person"
+  ) %}
 {% endif %}
+
+{{ onsSummary({
+  "withinQuestion": true,
+  "summaries": [
+    {
+      "groups": [
+        {
+          "groupTitle": list_title,
+          "headers": headers,
+          "rows": rows,
+          "summaryLink": {"url": add_link, "text": add_link_text, "dataQa": "add-item-link"},
```

Comment:
There is an issue with the Design System implementation of this summaryLink component - instead of `dataQa` it should take a key `attributes` that maps to html element attributes - for example:

```
"attributes": {
  "data-qa": "add-item-link"
}
```

We should wait for that change to merge and update this line accordingly before merging this change.

Reply:
Done, thanks!